compress.go

```diff
@@ -44,13 +44,13 @@ type flusher interface {
 	Flush() error
 }
 
-func (w *compressResponseWriter) Flush() {
+func (cw *compressResponseWriter) Flush() {
 	// Flush compressed data if compressor supports it.
-	if f, ok := w.compressor.(flusher); ok {
-		f.Flush()
+	if f, ok := cw.compressor.(flusher); ok {
+		_ = f.Flush()
 	}
 	// Flush HTTP response.
-	if f, ok := w.w.(http.Flusher); ok {
+	if f, ok := cw.w.(http.Flusher); ok {
 		f.Flush()
```

Comment:
We should check the error here

Reply:
Do you mean we should handle the false codnition of `ok` (bool)?

Reply:
The http.Flusher.Flush won't return the error. Do you mean I should check the false case of `ok` (bool)?

Reply:
Apologies, I did not look close enough at this and thought it was similar to the above f.Flush().